CDCgov/prime-reportstream
# Monkeypox demographics schema updates

This PR adds race, ethnicity, sex, and age to the dcipher schema and makes patient_state non-required.

Test Steps:
1. Process a sample monkeypox HL7 message, confirm subject_race, subject_ethnicity, subject_sex, and subject_age columns are present in the dcipher output.   ** Note: subject_age is not yet populating
3. Process a sample monkeypox HL7 message without pid-11.4, confirm message does not fail.

## Changes
- added subject_race, subject_ethnicity, subject_sex, and subject_age to dcipher-monkeypox.schema
- changed patient_state cardinality to ZERO_OR_ONE in monkeypox.schema

## Checklist

### Testing

## Linked Issues
- Fixes #6275
